### DISCUSSION TITLE
Master Warning MCP light ---> must be working

### DISCUSSION THREAD

**Discussion by CaptainJankez** [magknight/787aviatorsTracker#778] [Q&A] [answered]
So the reason why I write is an idea - to bring to life the master warn mcp button to life, it exist only when switching the lights mode to test, but it should be working for example, when engine goes on fire, when takeoff config is not correct, or other things that can cause the master warning alarm to turn on. I miss this feature so much, and I know that it can be made without waiting for systems. 

Cpt Jankez

**Comment by aeroniemi** [comment, marked as answer]
It does - there are master warnings (I think ~10 of them). We won't be adding those associated with engine failures/fires etc as they are associated only with **failure** scenarios, which we do not, and will not be supporting.
